### PR TITLE
#159291645 fix all eslint issues in javascript code and assign appropriate terms to res and req.

### PR DESCRIPTION
**Pull request by bukolabisuga**
#### What does this PR do?
This PR fixes all eslint issues with the javascript syntax in all javascript files
Improve code readability
Improve coveralls test coverage
Add necessary information to README.md

#### Description of Task to be completed?
Replace all for loops with appropriate es6 syntax equivalent
Replace all req and res shorthands with request and response
Ensure javascript code meets the eslint standard
Ensure all tests are passing
Improve README.md by adding necessary information and requirements

#### How should this be manually tested?
View travisCI badge in the repository README
View coveralls badge in the repository README
View License, Technologies used and other information in the repository README

#### What are the relevant pivotal tracker stories?
#159291645
#159291633
#159291622
#159291877

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
-import express from 'express';
+import expresponses from 'express';
```
Comment (fidelisojeah) [inline index.js:1]: `import express from 'express` was fine here though

File: server/controllers/EntryController.js [hunk 2]
```diff
@@ -1,49 +0,0 @@
-import Entry from '../entry';
```
Comment (fidelisojeah) [inline server/controllers/EntryController.js]: You deleted this entire file?? why?
This was the right way to go about it

File: server/routes.js [hunk 3]
```diff
@@ -1,19 +1,67 @@
...
-appRouter.get('/', (req, res) => res.send('Welcome to MyDiary API End Points!'));
+const entries = [
+  new Entry(1, 'My moms 50th birthday', 'Oh! What a lovely day it was to see her so happy'),
+  new Entry(2, 'My birthday', 'Happy birthday to meeeee'),
+  new Entry(3, 'My last day at school', 'I have waited for this day all my life'),
+  new Entry(4, 'My first bicycle', 'Your fave does not have a bike as cool as mine'),
+];
 
-appRouter.get('/api/v1/entries', EntryController.getEntries);
+appRouter.get('/', (request, response) => response.send('Welcome to MyDiary API End Points!'));
```
Comment (fidelisojeah) [inline server/routes.js]: Could you revert your changes on this?
Separate your routes from your controllers.

File: server/routes.js [hunk 4]
```diff
@@ -1,19 +1,67 @@
...
 
-appRouter.get('/api/v1/entries/:id', EntryController.getEntry);
+appRouter.get('/api/v1/entries', (request, response) => response.status(200).json({
+  success: 'success', message: 'Query successful', entry: entries,
+}));
 
-appRouter.post('/api/v1/entries', EntryController.addEntry);
+appRouter.get('/api/v1/entries/:id', (request, response) => {
```
Comment (fidelisojeah) [inline server/routes.js]: Same with this

File: server/routes.js [hunk 5]
```diff
@@ -1,19 +1,67 @@
...
+  if (!entryToGet || entryToGet === undefined) {
+    return response.status(404).json({
+      error: 'The entry you requested for must have been removed or has not been created',
+    });
+  }
+  response.status(200).json({ success: 'success', message: 'Query successful', entry: entryToGet });
+});
 
-appRouter.put('/api/v1/entries/:id', EntryController.editEntry);
+appRouter.post('/api/v1/entries', (request, response) => {
```
Comment (fidelisojeah) [inline server/routes.js]: And this

**Review by fidelisojeah — CHANGES_REQUESTED** [review]
I also want additional info on your readme.
Please add the expected input for each endpoint (if any) in the readme.

File: index.js (hunk 1 above)
Comment (bukolabisuga) [reply index.js:1]: This feedback has been implemented now.

**Pull request merged**